docs/src/content/docs/core/define-integration.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: defineIntegration
+description: Makes creating integrations easier, and adds a few goodies!
```

Comment:
```suggestion
A powerful wrapper around the standard Astro Integrations API. It allows us to provide extra hooks, functionality and best-practices when creating Astro Integrations.
```

Maybe it's time this got more specific?

Reply:
Yeah definitely! I ran out of ideas, I'll update it also in JSDoc